drivers/include/at86rf2xx.h

```diff
@@ -45,7 +45,7 @@ extern "C" {
 /**
  * @brief   Maximum possible packet size in byte
  */
-#define AT86RF2XX_MAX_PKT_LENGTH        (127)
+#define AT86RF2XX_MAX_PKT_LENGTH        (IEEE802154_MAX_FRAME_LEN)
```

Comment:
mhm, the naming seems somewhat inconsistent here it is `IEEE802154_MAX_FRAME_LEN` instead of `IEEE802154_FRAME_LEN_MAX` compare to above `IEEE802154_CHANNEL_MAX`. Or what's the rational?